addons/point_of_sale/models/pos_order.py

```diff
@@ -869,7 +869,7 @@ class ReportSaleDetails(models.AbstractModel):
 
 
     @api.model
-    def get_sale_details(self, date_start=False, date_stop=False, config_ids=False, session_ids=False):
+    def get_sale_details(self, date_start=False, date_stop=False, config_ids=False, session_ids=False, include_products=True):
```

Comment:
@nla-odoo What is the use of the include_products param still.  Can not we just remove it?